server/controllers/review.js

```diff
@@ -0,0 +1,63 @@
+const ObjectId = require("mongoose").Types.ObjectId;
+const asyncHandler = require("express-async-handler");
+const Profile = require("../models/Profile");
+const User = require("../models/User");
+const Review = require("../models/Review");
+
+// @route POST /review/:id
+// @Given parameters passed in, create a review.
+exports.createReview = asyncHandler(async (req, res, next) => {
+  const userId = req.body.id;
+  const profileId = req.params.id;
+  const reviewData = req.body;
+
+  if (!ObjectId(userId)) {
+    return res.status(400).send(Error("User id is invalid"));
+  }
+
+  try {
+    const { firstName, lastName, profileImg } = await User.findById(
+      userId
+    ).populate("profile");
+    const sitterProfile = await Profile.findById(profileId);
+    const newReview = new Review({
```

Comment:
This review isn't being saved at any point or linked to the sitter profile

Reply:
updated to save new review and sitter profile, and push new review id to sitter profile